test/puppetlabs/puppetdb/http/reports_test.clj

```diff
@@ -43,32 +55,42 @@
    ;; out of the example report object before comparison
    (-> report
        (update-in [:resource_events] (comp keywordize-keys munge-resource-events))
-       (update-in [:metrics] keywordize-keys))))
+       (update-in [:metrics] keywordize-keys)
+       strip-expanded)))
 
 (defn reports-response
+  "Convert expected results to a comparable format"
   [version reports]
-  (set (map report-response reports)))
+  (set (map report-response
+            reports)))
+
+(defn munge-actual-report
+  [report]
+  (-> report
+      (update-in [:resource_events] (comp keywordize-keys munge-resource-events))
+      strip-expanded))
```

Comment:
same comment with "actual" -- any action taken with factsets should be taken here